src/Redux/Categories/Categories.js

```diff
@@ -0,0 +1,17 @@
+const BOOK_STATUS = 'BOOKSTORE/Categories/Book_STATUS';
```

Comment:
Great job following the [Duck Pattern](https://github.com/erikras/ducks-modular-redux) 👍🏽

Reply:
Thanks, man 😃